docs/src/main/sphinx/release/release-450.md

```diff
@@ -0,0 +1,68 @@
+# Release 450 (7 Jun 2024)
+
+## General
+
+* Add support for the `optimizer.push-filter-into-values-max-row-count`
```

Comment:
Invert 

Add support to define the number of rows to push from a filter into a value with the xx prop